crates/sui-rpc-loadgen/src/payload/rpc_command_processor.rs

```diff
@@ -189,6 +220,13 @@ impl Processor for RpcCommandProcessor {
 
     async fn prepare(&self, config: &LoadTestConfig) -> Result<Vec<Payload>> {
         let clients = self.get_clients().await?;
+        if let CommandData::QueryTransactions(command_data) = &config.command.data {
+            if let Some(true) = command_data.from_file {
+                // TODO: how to generalize for other commands, flexibility beyond addresses
+                self.load_cache_from_file("addresses");
+            }
```

Comment:
Similar to `divide_checkpoint_tasks`, we need to distribute the task among different threads